ui/overview_page.go

```diff
@@ -551,7 +624,23 @@ func (pg *overviewPage) syncStatusTextRow(gtx layout.Context, inset layout.Inset
 	})
 }
 
-// syncBoxTitleRow lays out the progress bar.
+func (pg *overviewPage) syncStatusIcon(gtx layout.Context) layout.Dimensions {
+	syncStatusIcon := pg.notSyncedIcon
+	if pg.walletInfo.Synced {
+		syncStatusIcon = pg.syncedIcon
+	}
+	i := layout.Inset{Right: values.MarginPadding16, Top: values.MarginPadding9}
+	if pg.walletInfo.Syncing {
+		return i.Layout(gtx, func(gtx C) D {
+			return pg.syncingIcon.Layout(gtx)
+		})
+	}
+	return i.Layout(gtx, func(gtx C) D {
+		return syncStatusIcon.Layout(gtx, values.TextSize24)
```

Comment:
TextSize dimension shouldn't be used for margin or padding. Kindly define an equivalent of this dimension as MarginPadding